### PR TITLE
add assigner app

### PR DESCRIPTION
**Pull request by AndreyMlashkin**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -216,13 +216,17 @@
           default = all-gcc;
         };
         apps = {
+          assigner = {
+            type = "app";
+            program = "${self.packages.${system}.evm-assigner}/bin/assigner/assigner";
```
Comment (akokoshn) [inline flake.nix:221]: why `.evm-assigner`? executable is part of `zkevm-framework`

File: flake.nix [hunk 2]
```diff
@@ -216,13 +216,17 @@
...
+          };
           single-threaded = {
             type = "app";
             program = "${self.packages.${system}.proof-producer}/bin/proof-producer-single-threaded";
           };
           default = {
```
Comment (akokoshn) [inline flake.nix:227]: default option is required?

Comment (AndreyMlashkin) [reply flake.nix:227]: yes, a proof-producer script required it. I can change the script and remove the default, if you think, it's better

File: flake.nix (hunk 1 above)
Comment (AndreyMlashkin) [reply flake.nix:221]: this command works for me:
```
nix run  #assigner
```

I see, that the target is declared in zkevm-framework/tests/libs/assigner_runner/CMakeLists.txt, but it is deployed to the assigner folder...

Comment (AndreyMlashkin) [reply flake.nix]: I have corrected the package name now. I have no idea, why it was working before

File: flake.nix (hunk 2 above)
Comment (akokoshn) [reply flake.nix:227]: Yes, thanks!. it's a little bit strange when required build some  application i any case

Comment (AndreyMlashkin) [reply flake.nix]: fixed

**Review by akokoshn — APPROVED** [review]

**Pull request merged**